src/navigation-plugin.tsx

```diff
@@ -433,6 +451,10 @@ export class NavigationPlugin extends KalturaPlayer.core.BasePlugin {
     this._pluginButtonRef = ref;
   };
 
+  isPluginActive(): boolean {
+    return this._isPluginActive();
```

Comment:
it seems like we can make `this._isPluginActive` public and don't make one-more getter in plugin class

Reply:
done